test/test_ecs_utils.py

```diff
@@ -26,6 +26,33 @@ def mock_list_ecs_services(*, cluster):
     }
 
 
+def mock_list_ecs_tasks():
```

Comment:
This seems fine to have, though I'd rather it be part of a MockECSClient class that the MockBoto3Client could issue up. There is a mock GUID dispenser, and a fresh date could be generated on each call, and then this would be suitable for other uses. But I don't see a use of it getting introduced. Is that still coming? Neither `list_ecs_tasks` nor `run_ecs_task` seems to have unit tests.  I could do that generalization later, though. The functions seem simple enough and this is probably fine for now.